pages/_app.tsx

```diff
@@ -1,19 +1,22 @@
 import "../styles/globals.css";
-import { createTelemetryClient, TelemetryProvider } from "@lib/telemetry";
-import { Provider } from "next-auth/client";
-import type { AppProps } from "next/app";
+import AppProviders from "@lib/app-providers";
+import type { AppProps as NextAppProps } from "next/app";
 import Head from "next/head";
 
-function MyApp({ Component, pageProps }: AppProps) {
+// Workaround for https://github.com/zeit/next.js/issues/8592
+export type AppProps = NextAppProps & {
+  /** Will be defined only is there was an error */
+  err?: Error;
+};
+
+function MyApp({ Component, pageProps, err }: AppProps) {
   return (
-    <TelemetryProvider value={createTelemetryClient()}>
```

Comment:
where is Telementry now?

Reply:
I've created an `app-providers.tsx` file with all of them, and then in the `_app.tsx` we only keep the `<AppProviders />`.

Reply:
ah epic! much better solution